build-from-scratch.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
I don't think `build-from-scratch` is a good name. `build.sh` also build everything from scratch.
Maybe `build-without-powershell.sh`? I think name should emphasise what's different about this build script.